digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/queries.scala

```diff
@@ -368,4 +368,21 @@ object Queries {
   implicit object GetByteArray extends GetResult[Array[Byte]] {
     def apply(rs: PositionedResult) = rs.nextBytes()
   }
+
+  def mergeMunicipalities(municipalityToDelete: Int, municipalityToMerge: Int): Unit = {
+    sqlu"""UPDATE ASSET SET MUNICIPALITY_CODE = $municipalityToMerge, MODIFIED_DATE = SYSDATE, MODIFIED_BY = 'batch_process_municipality_merge' WHERE MUNICIPALITY_CODE = $municipalityToDelete;""".execute
+    sqlu"""UPDATE UNKNOWN_SPEED_LIMIT SET MUNICIPALITY_CODE = $municipalityToMerge WHERE MUNICIPALITY_CODE = $municipalityToDelete;""".execute
+    sqlu"""UPDATE INACCURATE_ASSET SET MUNICIPALITY_CODE = $municipalityToMerge WHERE MUNICIPALITY_CODE = $municipalityToDelete;""".execute
+    sqlu"""UPDATE INCOMPLETE_LINK SET MUNICIPALITY_CODE = $municipalityToMerge WHERE MUNICIPALITY_CODE = $municipalityToDelete;""".execute
+
+    // should be automatic, so maybe this query is not needed
+    // confirm this
+    sqlu"""UPDATE TEMP_ROAD_ADDRESS_INFO SET MUNICIPALITY_CODE = $municipalityToMerge WHERE MUNICIPALITY_CODE = $municipalityToDelete; """.execute
+
+    // Update on municipalityToMerge should be in daily batch so no need to update that here only delete the municipalityToDelete rows
+    // confirm this
+    sqlu"""DELETE FROM MUNICIPALITY_VERIFICATION WHERE MUNICIPALITY_ID = $municipalityToDelete;""".execute
```

Comment:
The load_municipalities_verification_info batch runs daily indeed. So the delete is the only thing you need.